website/docs/reference/dbt-classes.md

```diff
@@ -32,6 +34,8 @@ class Relation:
 
 ### Using relations
 
+Use relations in your SQL code by wrapping them in double curly brackets `{{ }}` (such as the [Jinja function {{ this }}](/reference/dbt-jinja-functions/this)). For example:
+
```

Comment:
```suggestion
In addition to `api.Relation.create`, dbt returns a Relation when you use [`ref`](/reference/dbt-jinja-functions/ref), [`source`](/reference/dbt-jinja-functions/source) or  [`this`](/reference/dbt-jinja-functions/this).
```

This isn't wrong, but it's not precise so I think it'll build an inaccurate mental model for people. It also conflicts with the code sample below, where the relation is created inside of a `{%  %} block not `{{ }}`.